meilisearch/models.py

```diff
@@ -0,0 +1,46 @@
+from dataclasses import dataclass
+from typing import Any, List, Dict, Optional
+from camel_converter.pydantic_base import CamelBase
+
+@dataclass
+class DocumentsResults:
+    results: List[Dict[str, Any]]
+    offset: int
+    limit: int
+    total: int
+
+class Task(CamelBase):
+    uid: Optional[str]
+    index_uid: str
+    task_uid: Optional[str]
+    status: str
+    type: str
+    details: Optional[Dict[str, Any]]
+    duration: Optional[str]
+    enqueued_at: str
+    started_at: Optional[str]
+    finished_at: Optional[str]
+
+class TaskInfo(CamelBase):
+    uid: Optional[str]
+    index_uid: str
+    task_uid: Optional[str]
```

Comment:
And `TaskInfo` should be used for every return from every asynchronous methods
```suggestion
class TaskInfo(CamelBase):
    task_uid: str
    index_uid: str
```